tools/graph_generation/grapher.r

```diff
@@ -0,0 +1,63 @@
+install.packages("ggplot2")
+library("ggplot2")
+library("plyr")
+
+setwd("C:\\Users\\tirpi\\Downloads")
+ue4_stats <- read.csv("ue4_stats.csv", row.names = NULL)
+acl_stats <- read.csv("acl_stats.csv", row.names = NULL)
+
+name <- factor(rep(c("ue4", "acl"), times = c(nrow(ue4_stats), nrow(acl_stats))))
+compression_ratio <- c(ue4_stats$Compression.Ratio, acl_stats$Compression.Ratio)
+max_error <- c(ue4_stats$Max.Error, acl_stats$Max.Error)
+
+bundle <- data.frame(name, compression_ratio, max_error)
+mu_ratio <- ddply(bundle, "name", summarise, grp.mean = mean(compression_ratio))
+mu_error <- ddply(bundle, "name", summarise, grp.mean = mean(max_error))
+
+p1 <- ggplot(ue4_stats, aes(x = Compression.Ratio)) +
+  geom_histogram(color = "black", fill = "white") +
+  geom_vline(aes(xintercept = mean(Compression.Ratio)), color = "blue", linetype = "dashed", size = 1)
+
+png(filename = "ue4_compression_ratio.png")
```

Comment:
might be worth checking to make sure the png generation settings enable compression or use the proper bit rate, not sure what the default settings are, //todo